### ISSUE TITLE
Not compatible with latest RC

### ISSUE THREAD

**Issue by ferdiemmen** [bibitoo/ng2-medium-editor#1]
Could you please update the repository to meet the new Release candidate version of Angular2?

**Comment by saiyaff** [comment]
Yeah.  I was checking this out. Wondering those 'directives' are no more there in the component.ts
Adding the editor component to the declarations would work?

**Comment by jaronwright** [comment]
Yes please
